fe/fe-core/src/main/java/org/apache/iceberg/hive/IcebergHiveCatalog.java

```diff
@@ -370,6 +373,27 @@ private void deletePath(String location) {
         }
     }
 
+    private void deleteTableFolder(FileIO io, TableMetadata metadata) {
```

Comment:
you also need to adapt the `deleteUncommittedDataFiles`

Reply:
done